roles/percona-server/tasks/logging.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: set up log rotation for mysql
-  logrotate: name=mysql path=/var/log/mysql/*.log
+  logrotate: name=mysql path=/var/log/mysql/*.err
```

Comment:
shouldnt this just be *? we need to get the mysql.log as well.
Or better yet two logrotate steps:
/var/log/mysql/*.log
/var/log/mysql/*.err